views/js/recommended-modules.js

```diff
@@ -142,7 +145,14 @@ var mbo = {};
         recommendedModulesRequest.done(function(data) {
           var recommendedModulesContainer = new RecommendedModulesContainer(config, data.content);
 
-          $(pageMap.contentContainer).append(recommendedModulesContainer.getMarkup());
+          let contentContainer;
+          if (pageMap.contentContainer instanceof Array) {
+            contentContainer = pageMap.contentContainer.find(element => $(element).length);
+          } else {
+            contentContainer = pageMap.contentContainer;
+          }
+
+          $(contentContainer).append(recommendedModulesContainer.getMarkup());
```

Comment:
```suggestion
          $(pageMap.contentContainer).first().append(recommendedModulesContainer.getMarkup());
```

So you could just do this with my previous comment